### PR TITLE
Latvian localization module addons/l10n_lv added

### PR DESCRIPTION
**Pull request by arnis-putnins**
Description of the issue/feature this PR addresses: Latvian localization module addons/l10n_lv added

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by arnis-putnins** [comment]
Hi, 2 weeks already passed, review please this PR.

File: addons/l10n_lv/data/account_account_tag_data.xml [hunk 1]
```diff
@@ -0,0 +1,1426 @@
...
+    <field name="color" eval="8"/>
+  </record>
+
+ <record id="account_account_tag_a_490" model="account.account.tag">
+    <field name="name">A.490. Asociēto sabiedrību parādi</field>
+    <field name="applicability">accounts</field>
+    <field name="color" eval="8"/>
+  </record>
+
+<record id="account_account_tag_a_500" model="account.account.tag">
```
Comment (h4818) [inline addons/l10n_lv/data/account_account_tag_data.xml:250]: The indentation is off, please check the entire file

File: addons/l10n_lv/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    """,
+    'license': 'LGPL-3',
+    'author': "Allegto IT, Chick.Farm",
+    'website': "https://allegro.lv",
+    'category': 'Accounting/Localizations/Account Charts',
+    'depends': [
+    'l10n_multilang',
+    ],
+    'data': [
+# Data for Trial balance and profit/loss report
```
Comment (h4818) [inline addons/l10n_lv/__manifest__.py:28]: Can remove comments

File: addons/l10n_lv/data/account.group.template.csv [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+"l10n_lv_group_78","78","Pārskata periodā iekļaujamās iepriekšējo periodu izmaksas",l10n_lv.account_chart_template_latvia
+"l10n_lv_group_79","79","Sociālās infrastruktūras uzturēšanas izmaksas",l10n_lv.account_chart_template_latvia
+"l10n_lv_group_8","8","Dažādi ieņēmumi un izmaksas, peļņa un zaudējumi",l10n_lv.account_chart_template_latvia
+"l10n_lv_group_81","81","Dažādi ieņēmumi",l10n_lv.account_chart_template_latvia
+"l10n_lv_group_82","82","Dažādas izmaksas",l10n_lv.account_chart_template_latvia
+"l10n_lv_group_83","83","Ārkārtas ieņēmumi",l10n_lv.account_chart_template_latvia
+"l10n_lv_group_84","84","Ārkārtas izmaksas",l10n_lv.account_chart_template_latvia
+"l10n_lv_group_86","86","Peļņa vai zaudējumi",l10n_lv.account_chart_template_latvia
+"l10n_lv_group_88","88","Nodokļi no peļņas un citi saimnieciskās darbības izdevumos neiekļaujamie nodokļi",l10n_lv.account_chart_template_latvia
+"l10n_lv_group_9","9","Papildus darba konti",l10n_lv.account_chart_template_latvia
```
Comment (h4818) [inline addons/l10n_lv/data/account.group.template.csv:61]: new line at end of file

File: addons/l10n_lv/data/account.account.template.csv [hunk 4]
```diff
@@ -0,0 +1,263 @@
+"id","name","code","account_type","chart_template_id/id","tag_ids/id","reconcile"
+"a1110","Pētniecības un uzņēmuma attīstības izmaksas",1110,"asset_non_current","l10n_lv.account_chart_template_latvia","","False"
```
Comment (h4818) [inline addons/l10n_lv/data/account.account.template.csv:2]: If possible, It is preferred to have english names for accounts and taxes (that are translated) as it makes it easier to maintain.

File: addons/l10n_lv/Readme.rtf [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (h4818) [inline addons/l10n_lv/Readme.rtf:1]: ?

File: addons/l10n_lv/data/account_tax_template_data.xml [hunk 6]
```diff
@@ -0,0 +1,499 @@
...
+    <!-- 21% VAT, VAT report R50 + PVN 1.2. -->
+       <record id="account_tax_template_sales_VAT_5012SG_tag" model="account.tax.template">
+        <field name="chart_template_id" ref="account_chart_template_latvia"/>
+        <field name="name">Iegāde 21% Latvijā</field>
+        <field name="description">21% PVN</field>
+        <field name="type_tax_use">purchase</field>
+        <field name="amount">21.0</field>
+        <field name="tax_group_id" ref="tax_group_vat_21"/>
+        <field name="amount_type">percent</field>
+        <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
```
Comment (h4818) [inline addons/l10n_lv/data/account_tax_template_data.xml:235]: AYH to check

File: addons/l10n_lv/views/res_partner_view.xml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_partner_form">
+        <field name="name">res_partner.form.title</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+                <field name="title" position="after">
```
Comment (h4818) [inline addons/l10n_lv/views/res_partner_view.xml:9]: what is intended here?
An l10n should avoid changing the partner form.

**Review by h4818 — COMMENTED** [review]
@arnis-putnins thanks for the contribution
I have done a quick preliminary review
a few comments about indentation and end of files
@jco-odoo @tsb-odoo

File: addons/l10n_lv/data/account_tax_template_data.xml [hunk 8]
```diff
@@ -0,0 +1,499 @@
...
+            }),
+            (0,0, {
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
+              'tag_ids': [ref('VAT_5012PG_tag')],
+            }),
+            (0,0, {
+              'factor_percent': -100,
+              'repartition_type': 'tax',
+              'account_id': ref('a5721'),
```
Comment (h4818) [inline addons/l10n_lv/data/account_tax_template_data.xml:248]: same account?

File: addons/l10n_lv/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+        co-author is Chick.Farm (visit for more information https://www.myacc.cloud)
+    """,
+    'license': 'LGPL-3',
+    'author': "Allegto IT, Chick.Farm",
+    'website': "https://allegro.lv",
+    'category': 'Accounting/Localizations/Account Charts',
+    'depends': [
+    'l10n_multilang',
+    ],
+    'data': [
```
Comment (h4818) [inline addons/l10n_lv/__manifest__.py:27]: please don't forget demo data

File: addons/l10n_lv/__manifest__.py [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+        author is Allegro IT (visit for more information https://www.allegro.lv)
+        
+        co-author is Chick.Farm (visit for more information https://www.myacc.cloud)
+    """,
+    'license': 'LGPL-3',
+    'author': "Allegto IT, Chick.Farm",
+    'website': "https://allegro.lv",
+    'category': 'Accounting/Localizations/Account Charts',
+    'depends': [
+    'l10n_multilang',
```
Comment (h4818) [inline addons/l10n_lv/__manifest__.py:25]: indent

File: addons/l10n_lv/views/res_partner_view.xml (hunk 7 above)
Comment (h4818) [reply addons/l10n_lv/views/res_partner_view.xml:9]: It seems this file can be removed

File: addons/l10n_lv/__manifest__.py (hunk 10 above)
Comment (h4818) [reply addons/l10n_lv/__manifest__.py:25]: 'account'

File: addons/l10n_lv/data/account.account.template.csv [hunk 11]
```diff
@@ -0,0 +1,263 @@
+"id","name","code","account_type","chart_template_id/id","tag_ids/id","reconcile"
```
Comment (h4818) [inline addons/l10n_lv/data/account.account.template.csv:1]: if no `tag_ids` are specified, the field can be removed

File: addons/l10n_lv/data/account_account_tag_data.xml [hunk 12]
```diff
@@ -0,0 +1,1426 @@
...
+    </record>
+
+<!-- – sākotnējā preču saņēmēja daļēja aizvietošana; -->
+     <record id="VAT_2E4_tag" model="account.account.tag">
+    <field name="name">Summa PVN2 E4</field>
+    <field name="applicability">taxes</field>
+    <field name="country_id" ref="base.lv"/>
+    </record>
+
+
```
Comment (h4818) [inline addons/l10n_lv/data/account_account_tag_data.xml:1424]: extra lines

File: addons/l10n_lv/Readme.rtf (hunk 5 above)
Comment (myacc-pro) [reply addons/l10n_lv/Readme.rtf:7]: updated in repository with description in english and latvian

File: addons/l10n_lv/__manifest__.py (hunk 10 above)
Comment (myacc-pro) [reply addons/l10n_lv/__manifest__.py]: added depends and demo configuration

File: addons/l10n_lv/__manifest__.py (hunk 9 above)
Comment (myacc-pro) [reply addons/l10n_lv/__manifest__.py:28]: added

File: addons/l10n_lv/__manifest__.py (hunk 2 above)
Comment (myacc-pro) [reply addons/l10n_lv/__manifest__.py]: removed

File: addons/l10n_lv/views/res_partner_view.xml (hunk 7 above)
Comment (myacc-pro) [reply addons/l10n_lv/views/res_partner_view.xml]: file removed

**Comment by arnis-putnins** [comment]
Contributor myacc-pro added to Contributor License Agreement.

**Comment by JOZI-Odoo** [comment]
Good morning @h4818 . 

I am trying to see if the pull was added to Odoo's standard. However, it doesn't seem that it is there.

Am I missing something, or is it still in the queue?

**Comment by arnis-putnins** [comment]
Hi, this is very outdated PR, the recent is https://github.com/odoo/odoo/pull/112159.
But since Odoo suggests to change the main language to English we are preparing the the next PR in next days.

**Comment by JOZI-Odoo** [comment]
Thank you for the update, Arnis.

It is good to know that the pull is moving to be introduced to the Odoo standard.

**Pull request closed without merging**